### ISSUE TITLE
Is this still in Development?

### ISSUE THREAD

**Issue by pumuckli** [80prozent/FaceShiftC4D#1]
Hi there,

i found your Plugin and it's awesome - are you planning to continue development for this one?

Greetings, Simon

**Comment by 80prozent** [comment]
Hi Simon

yes this is still under development.
I consider this a very early beta version, and much will change in future.
A proper Retargeting -System and preview-window will be added....

Here is a link for some video-tuts for the current version.
They are not very good and in German, but maybe it helps.

http://www.differentdesign.de/downloads/faceshiftc4d_tut.zip

Feedback / critic or feature-requests are allways very welcome.

cheers

Robin

----- Original Message -----

**Comment by pumuckli** [comment]
German is no problem, i already got it working on my machine (needed to connect the nodes by hand, but then it worked). What i would really need right now is the "load Data" feature, but it seams that it is not yet working at all, i can open a faceshift record file, but cannot add the keys to the timeline. Tried to fix this myself, but i soon realized that my phyton skills are not good enough at all... :)

General bugs i noticed: when you change your c4d layout (e.g. from normal to animation), the plugin closes and loses all its information, and i had a few crashes whem experimenting.

**Comment by 80prozent** [comment]
Hi

if German is no problem, we can write in german too ? but english is no problem either.

please update your Plugin again, cause i fixed the issue with the not created nodes.

i do not see the issue with the "load Data". 

I save a *.fsb and a *.wav in Faceshift.
I open C4D with and the FaceshiftData, and create a Faceshift-UserData-Object and some Retargeting (or just open a allready made c4d-Scene with a faceshift-rigg)
I import the *.fsb by using the "import Recording".
The Plugin shows you how many frames have been imported.
I make shure a FaceShift-userData Object is linked with the plugin. it doesnt matter if enabled or not.
I click "AddRecording" and the imported Frames are plotted to the linked FaceShift-UserData-Object.
You will not see the imported Keyframes, until you select the FaceShift-UserData-Object.
For me this works fine.

That the Plugin closes on Layout change is normal, it would only stay opened, if it is defined as part of the new Layout
to test this: 
open your startlayout, open the plugin, got to "windows"->"customisation"->"Save as Startlayout"
now switch to Animation-layout = plugin disappears
switch back to startup-layout = plugin reappears.

I just tried, and for me the Plugin keeps it information when reappearing.
Even if i close and reopen the plugin, the information is still there.

What c4d version are you on ?
when the plugin crashes, can you look into the c4d-output-console and tell me if a error is shown there?

cheers

Robin

----- Original Message -----

**Comment by pumuckli** [comment]
Klar, Deutsch kann ich sowieso besser - dachte nur man bleibt hier "international" :). Ich werd mir morgen früh gleich nochmal die neueste Version holen, dachte eigentlich ich hätte die aktuelle aber wer weiß... Sobald (oder sofern) das Plugin nochmal abstürzt, schreib ich hier natürlich den Error-Log rein. Auf jeden Fall schonmal Danke für die Rückmeldung!

Grüße,
Simon

(Ah und: C4d R14)

**Comment by 80prozent** [comment]
Hi Simon

Du hast recht, auf Github is englisch wohl besser. Hier meine email. 
80prozent@differentdesign.de
da können wir ungestört deutsch reden :)

Ja - du hattest die aktuellste version, bis ich um ca 19 uhr eine neue erstellt habe :)

Gruß

Robin

----- Original Message -----
